06-building-a-program-start-to-finish/delivering-your-product/README.md

```diff
@@ -0,0 +1,9 @@
+#### Output
+- **Using**: Medium
+- **Deliver**: a link to a post detailing your product and sharing your learning events/experiences
```

Comment:
@andela-kakpobome By 'some other problem the'd lid to talk about' you mean problems they encountered during execution?